.github/workflows/assign_new_prs_to_project.yml

```diff
@@ -11,3 +11,5 @@ jobs:
     uses: lawpilots/actions/.github/workflows/assign_to_project.yml@main
     secrets:
       MY_GITHUB_TOKEN: ${{ secrets.ORGA_GITHUB_TOKEN }}
+    with:
+      GH_PROJECT_URL: ${{ inputs.gh_project_url }}
```

Comment:
Here you actually need to specify the url (https://github.com/orgs/lawpilots/projects/3). 
assign_to_project.yml is the reusable workflow that is used by the other repos. But this workflow is for actually running this workflow on new PR's in the actions repo, so that they are added to the github project as well. So there are no inputs.